src/dmd/semantic3.d

```diff
@@ -479,37 +479,34 @@ private extern(C++) final class Semantic3Visitor : Visitor
 
             // Declare the tuple symbols and put them in the symbol table,
             // but not in parameters[].
-            if (f.parameterList.parameters)
+            if (f.parameterList.parameters) foreach (fparam; *f.parameterList.parameters)
```

Comment:
That `for` => `foreach`  transformation makes me a bit anxious, but according to the comment, it's valid 🤷‍♂ 
However I think we  usually  do:
```suggestion
            if (f.parameterList.parameters)
            foreach (fparam; *f.parameterList.parameters)
```

which IMO makes more  sense as the leftmost item is the most visible, and we want `foreach` to be more visible than a trivial `null` check.

Reply:
Done.